NMDSdevopsServiceAdm/DataEngineering
# Format import date in cleaning scripts

# Description
In each cleaning script, add a new column with the formatted import date

# How to test
Can be tested alongside https://trello.com/c/eTLzu2PQ/129-epic-create-a-generic-date-matrix-function-for-selecting-which-importdates-should-be-used-to-join-datasets-together-must

[Trello ticket](https://trello.com/c/Pj3tYAqr/175-create-new-column-with-a-formatted-import-date)

# Developer checklist
